static/styles/left_sidebar.css

```diff
@@ -190,25 +191,96 @@ li.show-more-topics {
     }
 }
 
-#private-container,
-#stream-filters-container {
+#left_sidebar_scroll_container {
+    outline: none;
     overflow-x: hidden;
     overflow-y: auto;
     position: relative;
     z-index: 0;
     width: 100%;
 }
 
-#stream-filters-container .simplebar-content-wrapper {
-    outline: none;
+#private_messages_container {
+    margin-bottom: $sections_vertical_gutter;
+    margin-right: 12px;
+
+    #toggle_private_messages_section_icon {
+        opacity: 0.7;
+        margin-left: 3px;
+        min-width: 13px;
+    }
+
+    #private_messages_section_header {
+        cursor: pointer;
+        padding-left: $far_left_gutter_size;
+        padding-right: 10px;
+        white-space: nowrap;
+
+        #show_all_private_messages {
+            right: 0;
+            float: right;
+            position: absolute;
+            opacity: 0.7;
+            text-decoration: none;
+            color: inherit;
+            margin-right: 25px;
+            margin-top: 1px;
+        }
+
+        .unread_count {
+            margin-right: 20px;
+            margin-top: 2px;
+        }
+    }
+
+    ul.pm-list {
+        list-style-type: none;
+        font-weight: 400;
+        margin-left: 0;
+        margin-bottom: 0;
+
+        span.fa-group {
+            font-size: 90%;
+        }
+
+        li.pm-list-item {
+            position: relative;
+            padding: 1px 10px 1px $far_left_gutter_size;
+            margin-left: 2px;
+
+            a {
+                text-decoration: none;
+                color: inherit;
+            }
+
+            .pm_left_col {
+                min-width: $left_col_size;
+            }
+        }
+
+        li.show_more_private_messages {
+            cursor: pointer;
+            padding-right: 30px;
+            padding-left: 16px;
```

Comment:
Same question with the 16 and 30 here -- can we extract some variables that show how they are related to other numbers in this file? 

This element is aligned with other stuff visually, but I can't make sense of where these numbers come from.